packages/react-components/styles/fonts.v2.tsx

```diff
@@ -0,0 +1,224 @@
+import colors from '@celo/react-components/styles/colors.v2'
+import { StyleSheet } from 'react-native'
+
+const Inter = {
+  Regular: 'Inter-Regular',
+  Medium: 'Inter-Medium',
+  SemiBold: 'Inter-SemiBold',
+}
+
+export const fontFamily = Inter.Regular
+
+const standards = {
+  large: {
+    fontSize: 17,
+    lineHeight: 24,
+    fontFamily: Inter.Regular,
+    color: colors.dark,
+  },
+  regular: {
+    fontSize: 15,
+    lineHeight: 20,
+    fontFamily: Inter.Regular,
+    color: colors.dark,
+  },
+  small: {
+    fontSize: 14,
+    lineHeight: 18,
+    fontFamily: Inter.Regular,
+    color: colors.dark,
+  },
+}
+
+export const fontStyles = StyleSheet.create({
+  h1: {
+    fontSize: 24,
+    lineHeight: 32,
+    fontFamily: Inter.Medium,
+    color: colors.dark,
+  },
+  h2: {
+    fontSize: 19,
+    lineHeight: 26,
+    fontFamily: Inter.SemiBold,
+    color: colors.dark,
+  },
+  sectionHeader: {
+    fontSize: 14,
+    lineHeight: 16,
+    fontFamily: Inter.Medium,
+    color: colors.dark,
+  },
+  label: {
+    fontSize: 13,
+    lineHeight: 16,
+    fontFamily: Inter.Regular,
+    color: colors.dark,
+  },
+  large: standards.large,
+  regular: standards.regular,
+  small: standards.small,
+  large600: { ...standards.large, fontFamily: Inter.SemiBold },
+  regular600: { ...standards.regular, fontFamily: Inter.SemiBold },
+  small600: { ...standards.small, fontFamily: Inter.SemiBold },
+  regular500: { ...standards.regular, fontFamily: Inter.Medium },
+
+  // paragraph: {
```

Comment:
would people find a transition object useful? where it had old names but the values were linked to the above new fonts? 
@tarikbellamine  @jeanregisser @i1skn 

otherwise I will remove

Reply:
This now has some common fonts style names in old system mapped to new font styles.